README.md

```diff
@@ -4,12 +4,28 @@ A open source screen reader for android.
 ## Build
 In development, choose the *debug build type*, because for the *release build type*, it's necessary to create a [keystore](https://developer.android.com/training/articles/keystore) and specify its properties in the `keystore.properties` file (use `keystore.properties.example` as a base).
 
-### IDE
-To build you need [Android Studio](https://developer.android.com/studio/releases) (Flamingo or later).
```

Comment:
Why you've deleted this? It's still true; we can eventually write that the latest version is required as we use modern build infrastructure and trying to follow latest stable versions of dependencies like AGP. In older versions of Android Studio the project may not work correctly or at all.

Reply:
Ok